lib/gather/cache.js

```diff
@@ -3,7 +3,10 @@ const getRepoInfo = require('git-repo-info');
 
 const gitInfo = getRepoInfo();
 
-const cachePath = `ember-codemods-telemetry-helpers-${gitInfo.sha}-${process.cwd()}`;
+const cwd = process.cwd().replace(':', '꞉');
```

Comment:
Hmm, instead of replacing with `:` we could use `path.parse(process.cwd()).dir` (with better formatting of course 😝).

What do you think?

Reply:
```bash
> path.parse(process.cwd()).dir
'C:\\Users'
```

Nope still returns `:`

If the purpose cache invalidation how about adding an unique identifier instead?